File: Android/MobileSeconndHand/app/src/main/java/marcin_szyszka/mobileseconndhand/models/NewAdvertisementItem.java
package marcin_szyszka.mobileseconndhand.models;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.Map;

/**
 * Created by marcianno on 2016-03-07.
 */
public class NewAdvertisementItem {
    public String AdvertisementTitle;
    public String AdvertisementDescription;
    public int AdvertisementPrice;
    public boolean IsOnlyForSell;
    public double Latitude;
    public double Longitude;
    public ArrayList<String> PhotosPaths;

    public NewAdvertisementItem(){
        PhotosPaths = new ArrayList<>();
    }
}

File: Xamarin_Android/MobileSecondHand/MobileSecondHand.Models/Advertisement/NewAdvertisementItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MobileSecondHand.Models.Advertisement {
	public class NewAdvertisementItem {
		public string AdvertisementTitle { get; set; }
		public string AdvertisementDescription { get; set; }
		public int AdvertisementPrice { get; set; }
		public bool IsOnlyForSell { get; set; }
		public double Latitude { get; set; }
		public double Longitude { get; set; }
		public List<string> PhotosPaths { get; set; }
	}
}
